perceval/converters/cqasm_converter.py

```diff
@@ -155,15 +137,34 @@ def _get_gate_inf(self, statement):
     def _convert_statement(self, statement, gate_index, optimized_gate_sequence):
         gate_name, controls, targets, parameter = self._get_gate_inf(statement)
 
+        # TODO: refactor
+        if gate_name == 'X90':
+            gate_name = 'rx'
+            parameter = np.pi / 2
+        elif gate_name == 'mX90':
+            gate_name = 'rx'
+            parameter = - np.pi / 2
+        elif gate_name == 'Y90':
+            gate_name = 'ry'
+            parameter = np.pi / 2
+        elif gate_name == 'mY90':
+            gate_name = 'ry'
+            parameter = - np.pi / 2
+
         if not controls:
-            circuit_template = _CQASM_1_QUBIT_GATES.get(gate_name, None)
-            if not circuit_template:
+            # working with 1 qubit gates
+            if gate_name.lower() in catalog.list():
+                if parameter is None:
```

Comment:
I think this is redundant with the name of the gate. It should just be ignored if the gate is not one of the defined gates with a parameter.